src/main/java/flaxbeard/immersivepetroleum/common/blocks/tileentities/PumpjackTileEntity.java

```diff
@@ -210,7 +209,10 @@ public void tick(){
 							active = true;
 						}
 					}
-					
+					if(active)
+					{
```

Comment:
Apart from the missing empty line above the `if(active)` and the bracket being on the same line its good

Reply:
Thats greate to hear, changed the file to match the requested coding style.